### ISSUE TITLE
Accessibility issue - menu is not accessible 

### ISSUE THREAD

**Issue by jesperingels** [shopware/platform#3610]
### PHP Version

8.2

### Shopware Version

6.5

### Expected behaviour

When using the TAB key to tab through the menu I expect the flyout submenu to open when I press the ENTER key. 
When pressing the ENTER key on a menu-item that has submenu items it should open the flyout navigation. Just like it does when you hover it with your mouse. 
Here you can see an example of an accessible menu: [accessible mega-menu](https://accessible-mega-menu.netlify.app/)

### Actual behaviour

When using the TAB key to tab through the menu and press the ENTER key on a (parent) menu item this opens the category page. 

### How to reproduce

- Go to [Shopware Demo](https://shopware6demo.firemultimedia.nl/)
- Press TAB untill you see the menu item 'Clothing' being highlighted
- Now press ENTER
- You are now taken to the Clothing category page
- It's impossible to navigate to the Women category page when using the keyboard (for example someone who is blind)

**Comment by MarcelSchmaeing** [comment]
Hi,

thx for your issue we will put it into our new evaluation process and have a look at it again.

**Comment by shopwareBot** [comment]
[public] Automated response: This issue is linked to the internal issue [https://shopware.atlassian.net/browse/NEXT-37895](https://shopware.atlassian.net/browse/NEXT-37895).

 

[created from NEXT-37895, comment 488730]

**Comment by shopwareBot** [comment]
[public]Thx for your report. This is something that we will tackle with a bigger epic to improve the accessibility in general. It should be fixed with 6.6.7.0 but latest with the release of 6.7.0.0



[created from NEXT-37895, comment 496251]
